crates/bevy_sprite/src/render/mod.rs

```diff
@@ -372,7 +373,7 @@ pub fn extract_sprites(
             extracted_sprites.sprites.extend(
                 slices
                     .extract_sprites(transform, entity, sprite, handle)
-                    .map(|e| (commands.spawn_empty().id(), e)),
+                    .map(|e| (commands.spawn(RenderFlyEntity).id(), e)),
```

Comment:
Again this seems like a very easy footgun to accidentally run into. Not that I have any better ideas at the moment other than maybe adding the component automatically in ExtractComponentPlugin.

Reply:
yeah, just as you mentioned on Discord, manual tagging can easily to leak entity in some cases.